app/ports/templates/ports/stats_faq.html

```diff
@@ -0,0 +1,76 @@
+    <hr>
+    <p>The MacPorts statistics collects and submits the following information to the server:</p>
+    <ul>
+        <li>
+            <strong>A unique identifier for your MacPorts installation</strong><br/>
+            This identifier is generated by <code>uuidgen</code> when you first
+            install the statistics service. We use this identifier to overwrite
+            statistics data you submitted earlier – we keep at most one set of
+            statistics per user per month. The identifier does not convey any
+            information about your system or location.
+        </li>
+        <li>
+            <strong>Version numbers and system properties</strong><br/>
+            The statistics service collects and submits the versions of MacPorts,
+            OS X, GCC and Xcode installed on your system. It also tells us what
+            your CPU architecture, platform and build target architecture are.
+        </li>
+        <li>
+            <strong>A list of your installed ports</strong><br/>
+            The statistics daemon collects and sends a list of the ports you have
+            installed containing the following information for each port:
+            <ul>
+                <li>name and installed version</li>
+                <li>whether the port is active or inactive</li>
+                <li>chosen variants</li>
+            </ul>
+        </li>
+    </ul>
+
+    <a name="collection-frequency"></a>
+    <h4>How Often Are Statistics Sent?</h4>
+    <hr>
+    <p>The statistics daemon runs once a week, overwriting the data your installation did submit earlier in the same
+        month, if any. If the daemon fails
+        to send the data, the error is silently ignored.</p>
+
+    <a name="why-statistics"></a>
+    <h4>Why Do You Need Statistics?</h4>
+    <hr>
+    <p>We'd like to improve our understanding of how MacPorts and its ports are
+        used. The statistics will help us determine which operating systems, build
+        architectures and Xcode versions we need to support and test to provide
+        a better overall experience for our users and less unexpected failures. The
+        stats will also help us understand how quickly new MacPorts releases or port
+        updates are adopted and how long we should wait before we can start using new
+        features. The variant statistics will hopefully reveal ports where variants
+        other than the default ones are popular choices and for which ports it might be
+        advisable to test non-standard variants when updating a port, or even which
+        variants should become defaults. We hope to use the variants data to build
+        binary packages for non-default, but popular variants in the future.</p>
+
+    <a name="opt-in"></a>
+    <h4>How Can I Submit Statistics?</h4>
+    <hr>
+    <p>To start submitting statistics, install the <a href="https://github.com/macports/macports-ports/tree/master/sysutils/mpstats">mpstats</a> port in
```

Comment:
We could replace the link (see below) and probably write the explicit command `sudo port install mpstats` to make it even easier for users to do it.

Reply:
Thank you, I was about to add that.